cdap-ui/app/cdap/components/Card/Card.less

```diff
@@ -15,10 +15,16 @@
  */
 
 @import '../../styles/variables.less';
+
+@card-background-color: @cdap-orange;
+@card-font-color: #ffffff;
+
 .cask-card {
-  background: #fff;
-  border-radius: 4px;
+  background: @cdap-orange;
```

Comment:
This will affect every single card. We style card for splashscreen. Please namespace style for splash screen. Currently this is what we see 
![screen shot 2016-09-24 at 5 00 40 pm](https://cloud.githubusercontent.com/assets/1452845/18811976/752dd9e6-8278-11e6-9ee1-18a60aa28f66.png)